### PR TITLE
my side of the story

### PR DESCRIPTION
**Pull request by LisaALoman**
initialized a possible plot

### CODE REVIEW

File: fairytale.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ The average Squonk is 7 meters tall and screams loud.
 >  What happened between it and the *Drowsloks*?
 
 `Well... your turn now.`
+My story:
```
Comment (floridop) [inline fairytale.md:19]: Hi @LisaALoman ! Could you please move your story above the line
```
>  Add some lines to the story. Can be anything. Inspiration?  
```
that is, around line 10?

More info will come later.

**Comment by LisaALoman** [comment]
It is now moved up to line 10.

/ Lisa

**Comment by floridop** [comment]
Hi! 

I think you made the changes in your local copy but forgot to add , commit or push.
I do not see them in your branch...

The changes must show up here:
https://github.com/floridop/MNXB01-fairytale/pull/6/files

Cheers,
Florido

**Comment by LisaALoman** [comment]
Probably forgot that. I think it is fixed now.

**Comment by floridop** [comment]
thanks, that looks good for now. I will come with the final assignment after I process your colleague's work.

**Comment by LisaALoman** [comment]
The conflicts are resolved! Was there anything else I needed to do?

**Comment by floridop** [comment]
Grade updated on canvas, no need to do anything else, thanks!

**Pull request merged**